docs/DeveloperGuide.md

```diff
@@ -654,3 +655,28 @@ Team size: 5
 1. **Add more precise functionality to the `clean` command**. The `clean` command currently does not allow removal of 
 students who have graduated in the current year, as it can only detect the graduation year but not the month. We plan to 
 add support for storing a more specific graduation date, such that we can accurately remove students who have graduated immediately after their graduation.
+
+
+## **Appendix: Effort**
+
+### Overview
+
+As we have adapted AB3 for university dorm managers, our main efforts were in adding support for other necessary fields, enhancing the duplicate handling and data validation, and providing extra functions to streamline data saving, adding, updating and safeguard against mistakes.
+This posed substantial difficulties for us, as we had to work within the AB3 model and implement the multiple features to be compatible with the rest of the app. 
+
+Here are some of the achievements of DorManagerPro:
+* Fields
+  * Added fields for room number, emergency contacts and graduation year
+  * Implemented relevant duplicate handling and field constraints for room number, emergency contacts and graduation years.
+  * More specific and relevant field constraints and duplication handling for name, phone number, email address and tags
```

Comment:
Can we add full stops? Also, idk if this is very necessary but the correct punctuation for commas is "a, b, and c", instead of "a, b and c". Can we add the extra comma?